src/test/java/org/openrewrite/java/migrate/ReplaceLocalizedStreamMethodsTest.java

```diff
@@ -0,0 +1,100 @@
+package org.openrewrite.java.migrate;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class ReplaceLocalizedStreamMethodsTest implements RewriteTest {
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new ReplaceLocalizedStreamMethods("com.test.Runtime1 getLocalizedInputStream1(java.io.InputStream)", "com.test.Runtime1 getLocalizedOutputStream1(java.io.OutputStream)"));
+    }
+
+    //language=java
+    String LocalizedStreamMethodsClass = """
```

Comment:
When a class only supports the tests but should not itself be evaluated by recipes, then it's clearer to use the `.dependsOn` you can see applied here:
https://github.com/openrewrite/rewrite-spring/blob/729164fd713cbd0bfce44e582b0d613f4d1965b0/src/test/java/org/openrewrite/java/spring/test/SpringRulesToJUnitExtensionTest.java#L34****